react-blocks/src/styles/blocks/Media.scss

```diff
@@ -0,0 +1,9 @@
+#editor .editor-writing-flow > div > div > div > .editor-block-list__layout > div[data-type="planet4-blocks/media-video"] {
```

Comment:
this will break up blocks previews, some blocks will be wider than others so it could be confusing. let's not use it for now